### PR TITLE
{lib}[GCCcore/8.2.0] libspatialindex v1.8.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3135.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/b97c9560cc70e69b7de3f5f61458db14 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2485.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e84d072ac5eea29783f1f6bafcb134cc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2084.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9318bdecaa272bb78505f0213cabc525 for a full test report.

File: easybuild/easyconfigs/l/libspatialindex/libspatialindex-1.8.5-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'libspatialindex'
+version = '1.8.5'
+
+homepage = 'http://libspatialindex.github.io'
```
Comment (Micket) [inline easybuild/easyconfigs/l/libspatialindex/libspatialindex-1.8.5-GCCcore-8.2.0.eb]: https

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/daa9ac569bb2e7da6a0eb66d41b28a37 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/00e81561bab31d0c5510f31024955271 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**